src/excursion/Excursion.svelte

```diff
@@ -0,0 +1,105 @@
+<script>
+  import { excursion } from "../store";
+  import { Collection } from "sveltefire";
+  const addToArray = (provisionRef) => {
+    provisionRef.add({});
+  };
+
+  const deleteFromArray = (provisionRef, id) => {
+    provisionRef
+      .doc(id)
+      .delete()
+      .then((r) => console.log(r));
+  };
+</script>
+
+<div>
+  <Collection
+    path={$excursion.collection('provisions')}
+    let:data={excursionsData}
+    let:ref>
+    <form on:submit|preventDefault={() => addToArray(ref)}>
+      <ul class="divide-y divide-gray-200">
+        {#each excursionsData as item}
```

Comment:
it doesn't look like you're using the actual object anywhere, so you can restructure it in line.

also I think it's always good to use the key when #eaching....
```html
{#each excursionsData as { price, id, currency } (id)}

```

Reply:
If a key expression is provided — which must uniquely identify each list item — Svelte will use it to diff the list when data changes, rather than adding or removing items at the end. The key can be any object, but strings and numbers are recommended since they allow identity to persist when the objects themselves change.

https://svelte.dev/docs#each